### DISCUSSION TITLE
How to stop a tensorboard server?. How to open a new tab for the same server?

### DISCUSSION THREAD

**Discussion by cezannealves** [microsoft/vscode-jupyter#15803] [Questions and Answers]
Currently, every time I launch tensorboard inside VSCode it starts a new server, piling up servers and reserved ports inside VSCode.

So I wanted to know, once I start a tensorboard server inside vscode using the [Tensorboard extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.tensorboard),
1- How can I open a new tab connecting to this server without starting a new server instance?
2- How can I stop Tensorboard server instance started by this extension

**Comment by mschossler** [comment]
Also interested in this same question.
